### DISCUSSION TITLE
VS code Karavan extension

### DISCUSSION THREAD

**Discussion by saravanakumar1987** [apache/camel-karavan#774] [Q&A]
Creating a new integration always keep loading. I tried re-installing the extension in VS code but same result. Any issues ?

<img width="776" alt="image" src="https://github.com/apache/camel-karavan/assets/10127910/d08f8c8f-0a0d-4a0c-8031-2949692b446c">

**Comment by mgubaidullin** [comment]
Look like it is solved https://camel.zulipchat.com/#narrow/stream/322293-camel-karavan/topic/.E2.9C.94.20VS.20code.20Karavan.20extension/near/360947709
